### ISSUE TITLE
Make pythonDot a little brighter for better contrast

### ISSUE THREAD

**Issue by pulcinello** [franbach/miramare#6]
First, thanks for this eye-candy theme. I really like it, but sometimes I find it's a little hard to distinguish the dot from normal space when editing python files. I'd appreciate if you can make it a little brighter.
![Screen Shot 2020-05-15 at 3 45 07 PM](https://user-images.githubusercontent.com/3975367/82025612-2a8f1d00-96c4-11ea-9bf6-0920b1ddd9a1.png)

**Comment by franbach** [comment]
Hi @pulcinello, you welcome! I'm going to fix this as soon as I can! Feel free to send me a PR.
Thanks!

**Comment by arnavb** [comment]
@franbach I'm noticing the same issue for JS. Has it been updated or no? I would send a PR but I'm not experienced at all with vim color schemes.
<img width="137" alt="Screen Shot 2020-12-20 at 1 40 41 PM" src="https://user-images.githubusercontent.com/30810402/102721467-f5395600-42c8-11eb-86cd-0ce9fbe14635.png">

Ok I changed it to white by using `highlight jsDot ctermfg=white` in my vimrc which seems to look good and I noticed there's a line setting it to grey in the vim file in this repo. Would fixing this be as simple as changing the color to white or would using a darker color (than white) be better?
